### PR TITLE
Add contribution notice

### LINKED ISSUE #94: Add contribution notice
Add contribution notice.

Contributors need to be made aware of the terms and conditions under which they contribute to this repository.

### PR DESCRIPTION
**Pull request by mfellner**
Closes #94.

### CODE REVIEW

**Comment by mfellner** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -55,8 +55,12 @@ apm install nuclide
 npm install -g flow-bin
 ```
 
+## Contributing
+
 For details on pull requests, commit messages and contributing please see [CONTRIBUTING.md](CONTRIBUTING.md)
 
+Unless you explicitly state otherwise in advance, any non trivial contribution intentionally submitted for inclusion in this project by you to the steward of this repository (Zalando SE, Berlin) shall be under the terms and conditions of the [MIT License](LICENSE), without any additional copyright information, terms or conditions.
+
```
Comment (Adeynack) [inline README.md]: Do we want that paragraph in BOTH files (`CONTRIBUTIN.md` and `README.md`)?

Comment (mfellner) [reply README.md]: I followed the recommendations from another Zalando OS repository. Technically we don't need the notice here as well, CONTRIBUTING.md should be enough.

**Comment by mfellner** [comment]
👍

**Comment by danpersa** [comment]
👍

**Pull request merged**